server/game/cards/08.6-SAT/GourdinDeBoisFlotte.js

```diff
@@ -0,0 +1,33 @@
+const DrawCard = require('../../drawcard.js');
+
+class GourdinDeBoisFlotte extends DrawCard {
+    setupCardAbilities(ability) {
+        this.attachmentRestriction({ trait: 'Drowned God' });
+
+        this.whileAttached({
+            effect: ability.effects.dynamicStrength(() => this.controller.deadPile.length)
+        });
+
+        this.interrupt({
+            when: {
+                onCharacterKilled: event => event.cardStateWhenKilled === this.parent
+            },
+            target: {
```

Comment:
This needs an added `type: 'select'` so the opponent won't get informed of the new parent before deciding whether to cancel.